1-exercises/E-while-loop-with-array/exercise.js

```diff
@@ -18,6 +18,11 @@ const BIRTHDAYS = [
 
 function findFirstJulyBDay(birthdays) {
     // TODO
+    for (let i = 0 ; i < BIRTHDAYS.length; i++)
+    if( BIRTHDAYS[i] ===  "July 11th" ) {
```

Comment:
Hi, for this question, I think you could change the condition so that you are checking for any date in July, rather than looking for only the 11th of July. So something like
// if (BIRTHDAYS[i].includes("July")) {
// return BIRTHDAYS[i];
// }

Reply:
Thank you for the help, I didn't really know about (includes) Jack just mentioned it and I googled it